corpora/csip/metadata/amdsec/CSIP32/test-case.xml

```diff
@@ -1,23 +1,88 @@
+    The requirement text cites METS and mentions embedded or linked metadata, but according to http://earkcsip.dilcis.eu/#534-use-of-the-mets-administrative-metadata-section-element-amdsec    .
+    CSIP is only using referencing of files containing administrative metadata. So embedding is not allowed.
+  </description>
   <!-- List of requirments that this test case depends on in addition to the
   main requirememt, e.g. general requirments on the form of IDs -->
   <dependencies>
+    <dependency requirementId="CSIPSTR6"  URL="http://earkcsip.dilcis.eu/#41-folder-structure-of-the-csip"/>
+    <dependency requirementId="CSIPSTR31" URL="http://earkcsip.dilcis.eu/#534-use-of-the-mets-administrative-metadata-section-element-amdsec"/>
   </dependencies>
   <!-- A list of the validation rules derived from the test case -->
   <rules>
+
+    <rule id="1">
+      <!-- Text description of the validation rule -->
+      <description>There should be an XML element named digiprovMD in the mets.xml</description>
+      <!-- Details of the validation error generated by the message, can be one of: [ ERROR | WARNING | INFO ]-->
+      <error level="WARNING">
+        <!-- Expected error message when the rule is violated. -->
+        <message>The element mets/amdSec/digiprovMD does not exist in METS.xml. It SHOULD be there.</message>
+      </error>
+      <corpusPackages>
+        <!-- Details of corpus packages designed to test this validation rule. -->
+        <package name="IP_18000_CSIP32_1" isValid="FALSE">
+          <path>/corpora/metadata/amdSec/CSIP32/IP_18000_CSIP32_1</path>
+          <!-- Full description of corpus package features. -->
+          <description>A package based on the minimal IP with schemas. Contains no digiprovMD</description>
+        </package>
+      </corpusPackages>
+    </rule>
+
+    <!-- Rule for consistency between digiprovMD and files in the metadata/descriptive folder - can only be a warning due to (too) loose requirement -->
+    <rule id="2">
+      <!-- Text description of the validation rule -->
+      <description>When one or more mets/amdSec/digiprovMD element(s) exist(s) in mets.xml there SHOULD be files in the folder metadata/preservation.
+        The mets schema allows no files to be present since it allows no href to be present in digiprovMD</description>
+      <!-- Details of the validation error generated by the message, can be one of: [ ERROR | WARNING | INFO ]-->
+      <error level="WARNING">
+        <!-- Expected error message when the rule is violated. -->
+        <message>One or more mets/amdSec/digiprovMD elements exist(s). No file(s) found in metadata/preservation. File(s) SHOULD be there.</message>
+      </error>
+      <corpusPackages>
+        <!-- Details of corpus packages designed to test this validation rule. -->
+        <package name="IP_18000_CSIP32_2" isValid="FALSE">
```

Comment:
isValid should be TRUE according to the new definition in the schema for warnings